### PR TITLE
cpuid, free, reboot, reset: add Hindi translation

### PR DESCRIPTION
**Pull request by lucifertrj**
Added `reset.md`, `reboot.md` and `cpuid.md` commands details in hindi.

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by suyashbhawsar** [comment]
Hello, it'd be better if you create separate pull requests for each page.

**Comment by blueskyson** [comment]
Thanks for contribution. You can keep working on this pull request. Please sign the [Contributor License Aggrement](https://cla-assistant.io/tldr-pages/tldr?pullRequest=8815) and fix the syntax errors.

File: pages.hi/linux/free.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+`free`
+ 
+- सिस्टम मेमोरी को बाइट्स/केबी/एमबी/जीबी में दिखाएं: 
+
+`free -{{b|k|m|g}}` 
+
+- मानव-पठनीय इकाइयों में सिस्टम मेमोरी प्रदर्शित करें: 
+
+`free -h'
```
Comment (suyashbhawsar) [inline pages.hi/linux/free.md:16]: Use this line instead `free -h`

**Review by suyashbhawsar — CHANGES_REQUESTED** [review]

File: pages.hi/linux/free.md (hunk 1 above)
Comment (suyashbhawsar) [inline pages.hi/linux/free.md:16]: Check this line, you've used quote instead of a backtick

**Review by suyashbhawsar — CHANGES_REQUESTED** [review]

**Comment by lucifertrj** [comment]
> Thanks for contribution. You can keep working on this pull request. Please sign the [Contributor License Aggrement](https://cla-assistant.io/tldr-pages/tldr?pullRequest=8815) and fix the syntax errors.

Done.

**Comment by suyashbhawsar** [comment]
Check if you have any whitespaces at the blank new lines or after any sentences

**Comment by lucifertrj** [comment]
I fixed the whitespace bug. And also agreed to CLA License

Screenshot from 2022-10-08 22-15-32
Screenshot from 2022-10-08 22-15-04

**Comment by blueskyson** [comment]
It seems that your CLA status is still pending, could you try to sign again and check the github account is correspond to your local git? (I used to face this situation when my user.email was wrongly set)

**Comment by lucifertrj** [comment]
IMG_20221008_234927

I have tried multiple times. I checked my config details, but still it is showing pending. When I open CLA link it is redirecting me back to GitHub.

File: pages.hi/linux/cpuid.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# cpuid
+
+> सभी सीपीयू के बारे में विस्तृत जानकारी दिखाएं।
+> अधिक जानकार: <http://etallen.com/cpuid.html>.
```
Comment (navarroaxel) [inline pages.hi/linux/cpuid.md:4]: We use this standard translation
```suggestion
> अधिक जानकारी: <http://etallen.com/cpuid.html>.
```

**Review by navarroaxel — APPROVED** [review]

**Review by blueskyson — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Comment by lucifertrj** [comment]
> LGTM, Thanks for your contribution.

You forgot to add Hacktoberfest label

**Comment by navarroaxel** [comment]
@lucifertrj I can add it, but I think it's not required :think

**Comment by kbdharun** [comment]
It isn't necessary to add the `hacktoberfest` label separately as our repository already has `hactoberfest` in its topic. This PR will soon be reflected in your profile. Thanks for your contribution.

**Comment by suyashbhawsar** [comment]
> @lucifertrj I can add it, but I think it's not required :thinking: 

Yup it's not required

**Comment by suyashbhawsar** [comment]
> > LGTM, Thanks for your contribution.
> 
> 
> 
> You forgot to add Hacktoberfest label

Your PR is going to be counted as 1/4 so, I'd recommend you to create different PRs for different pages

**Pull request merged**